entity-service/src/test/kotlin/com/egm/stellio/entity/service/EntityServiceTests.kt

```diff
@@ -12,7 +12,10 @@ import com.egm.stellio.shared.model.ResourceNotFoundException
 import com.egm.stellio.shared.model.parseToNgsiLdAttributes
```

Comment:
did not look at all the details, but not sure all the new tests make sense.

Reply:
before, for example, when we did addGeoPropertyToEntity, we had set location directly whereas now we can have something else. We had `subject.location` and now `subject.${'$'}propertyKey`